banzai/data.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, data: Union[np.array, Table], meta: fits.Header,
                  mask: np.array = None, name: str = '', uncertainty: np.array = None, memmap=True):
         super().__init__(data=data, meta=meta, mask=mask, name=name, memmap=memmap)
         if uncertainty is None:
-            uncertainty = self.read_noise * np.ones(data.shape, dtype=data.dtype) / self.gain
+            uncertainty = self.read_noise * self.n_sub_exposures * np.ones(data.shape, dtype=data.dtype) / self.gain
```

Comment:
it seems that we are assuming that site software, after co-adding images, is not messing with the readnoise keyword. Thinking more about this - this is a possibly messy interdependence between data acquistion and processing that needs future coordination between those two processes.

If I read code correctly:
* if there is no per pixel noise mask, we init the per pixel noise map with teh read noise. 
* when e,g, c = a+b, then error (c) = sqrt (error(a) **2 _ error (b)**2).  I.e., should we not multiply with sqrt (self.n_sub_exposures) ?
* Then also, if per pixel noise image exists, shoudl that not be also multiplied here?

Reply:
https://github.com/LCOGT/banzai/pull/392#discussion_r1722117589 @drhaz Yes this indeed messy. This is due to legacy cameras that did not have an overscan region. For old kb cameras we estimate the mean bias level using the whole image. For cameras that have them we include the overscan. The original thinking was if there is an overscan, then bias_level is going to be very close to zero so just don't worry about it. Things get messier thinking about stacked images though so I can think about that a little more.